src/test/regress/Makefile

```
@@ -43,7 +43,7 @@ output_files := $(patsubst $(citus_abs_srcdir)/output/%.source,expected/%.out, $
 # intermediate, for muscle memory backward compatibility.
 check: check-full
 # check-full triggers all tests that ought to be run routinely
-check-full: check-multi check-multi-mx check-multi-1 check-worker check-operations check-follower-cluster check-isolation check-failure
+check-full: check-multi check-multi-mx check-multi-1 check-worker check-operations check-follower-cluster check-isolation check-failure check-columnar check-columnar-isolation check-pg-upgrade check-arbitrary-configs check-citus-upgrade check-citus-upgrade-mixed check-citus-upgrade-local check-citus-upgrade-mixed-local
```

This is getting harder to read. How about either (i) writing a regex matching all possible `check-*` targets, or (ii) splitting this into multiple lines?

(i)
We seem to add all the targets in the form `check-*` unless they contain at least one of the following substrings: `vg`, `minimal`, `base`, `internal`, `full`, `hyperscale`, `vanilla`.

I think we should also add hyperscale and vanilla tests to this dependency list.

(ii)
A possible solution:

```suggestion
check-full: check-arbitrary-configs
check-full: check-citus-upgrade check-citus-upgrade-local check-citus-upgrade-mixed check-citus-upgrade-mixed-local check-pg-upgrade
check-full: check-columnar check-columnar-isolation
check-full: check-failure
check-full: check-follower-cluster
check-full: check-isolation
check-full: check-multi check-multi-1 check-multi-mx
check-full: check-operations
check-full: check-worker
```

------

I like the first option more, because
- if we introduce some new make targets, they will be captured as well
- it is easier to understand which targets are included this way
- it will be easier to maintain this file on enterprise repo

---

On second thought, AFAIK GNU Make does not support regex matching for phony dependencies, and it may not be as easy as it sounds to implement (i)